### PR TITLE
feature: Move gitlab configuration to config file CY-1353

### PR DESCRIPTION
**Pull request by jllopes**

### CODE REVIEW

File: docs/configuration/git-providers/gitlab-cloud.md [hunk 1]
```diff
@@ -1 +1,33 @@
...
+
+1.  Create an application pointing to your local Codacy deployment URL.
+
+    You'll need to add the following 'Redirect URI'. Make sure to update your protocol to use either http or https and your domain name as well. Keep in mind this field is case sensitive.
+
+    `https://codacy.example.com/login/GitLab`
+
+    `https://codacy.example.com/add/addProvider/GitLab`
+
+    `https://codacy.example.com/add/addService/GitLab`
```
Comment (rtfpessoa) [inline docs/configuration/git-providers/gitlab-cloud.md:15]: This is wrong, please check the docs on zendesk, I fixed it there

Comment (rtfpessoa) [reply docs/configuration/git-providers/gitlab-cloud.md:15]: Check for the right ones https://support.codacy.com/hc/en-us/articles/360012368539-Configuring-GitLab-com-with-Codacy-Self-hosted

Comment (rtfpessoa) [reply docs/configuration/git-providers/gitlab-cloud.md:15]: I would also put them all in the same block to copy easier together

File: docs/configuration/git-providers/gitlab-cloud.md [hunk 2]
```diff
@@ -1 +1,33 @@
 # GitLab Cloud
+
+## GitLab Application
+
+Follow the instructions below to set up the Codacy integration with GitLab Cloud:
+
+1.  Create an application pointing to your local Codacy deployment URL.
```
Comment (pauloribeiro-codacy) [inline docs/configuration/git-providers/gitlab-cloud.md:7]: Can we provide info on how to create the GitLab Application? If we can provide the direct URL to the Applications screen on GitLab it should be enough:

```suggestion
1.  Create a [new application](https://gitlab.com/oauth/applications) on GitLab pointing to your local Codacy deployment URL.
```

Comment (rtfpessoa) [reply docs/configuration/git-providers/gitlab-cloud.md:7]: We should have the same experience we had in the previous docs. I placed the link in a previous comment

Comment (pauloribeiro-codacy) [reply docs/configuration/git-providers/gitlab-cloud.md:7]: @rtfpessoa is right, we're also missing the scopes that must be checked, for instance. :+1:

File: docs/configuration/git-providers/gitlab-cloud.md [hunk 3]
```diff
@@ -1 +1,34 @@
 # GitLab Cloud
+
+## GitLab Application
+
+Follow the instructions below to set up the Codacy integration with GitLab Cloud:
+
+1.  Create a [new application](https://gitlab.com/profile/applications) pointing to your local Codacy deployment URL with `api`, `read user` and `read repository` scopes.
+
+    You'll need to add the following 'Redirect URI'. Make sure to update your protocol to use either http or https and your domain name as well. Keep in mind this field is case sensitive.
```
Comment (rtfpessoa) [inline docs/configuration/git-providers/gitlab-cloud.md:9]:
```suggestion
    You'll need to add the following 'Redirect URI'. Make sure to update your protocol to use either http or https and your hostname as well. Keep in mind this field is case sensitive.
```

**Pull request merged**